<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoWidget</class>
 <widget class="QWidget" name="DemoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demonstration</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>730</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFaster">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Higher rate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSlower">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>730</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Lower rate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>730</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="dispCurve" native="true">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>0</y>
     <width>521</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>660</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMax">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Band
edge</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValues">
   <property name="geometry">
    <rect>
     <x>725</x>
     <y>348</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Значения:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>22</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>720</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelMassValue">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>730</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMass">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>720</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Текущая частота</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/lambda.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelW">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>438</y>
     <width>32</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Текущая частота</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'palatino linotype,new athena unicode,athena,gentium,code2000,serif'; font-size:16pt; &quot;&gt; ω =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOmega">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>438</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Текущая частота</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelKValue">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>468</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelK">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>468</y>
     <width>30</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;font-family:'palatino linotype,new athena unicode,athena,gentium,code2000,serif'; font-size:14pt;&quot;&gt; K =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMinAcoustic">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Lower
bound</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMaxAcoustic">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>390</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Upper
bound</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMaxOptical">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Upper
bound</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameWindow">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Uniform Chain&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>730</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelW0">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>640</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'serif'; font-size:14pt; font-weight:600; font-style:italic; color:#000000;&quot;&gt;ω&lt;/span&gt;&lt;span style=&quot; font-family:'serif'; font-size:14pt; font-weight:600; font-style:italic; color:#000000; vertical-align:sub;&quot;&gt;o&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic;&quot;&gt;- natural frequency of the cell&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;img src=&quot;:/images/omega.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChastota">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Frequency&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHuyeybl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>540</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Optical branch&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAcMaxFormula">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>440</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/omegaAcoustMax.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOptMinFormula">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>310</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/omegaOpticalMin.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOptMaxFormula">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/omegaOpticalMax.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHuyeybl_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>580</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Acoustic branch&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>resetButton</zorder>
  <zorder>labelW</zorder>
  <zorder>labelOmega</zorder>
  <zorder>labelKValue</zorder>
  <zorder>labelK</zorder>
  <zorder>back</zorder>
  <zorder>pushButtonFaster</zorder>
  <zorder>pushButtonSlower</zorder>
  <zorder>pushButtonPause</zorder>
  <zorder>plot</zorder>
  <zorder>dispCurve</zorder>
  <zorder>graphicsView</zorder>
  <zorder>pushButtonMax</zorder>
  <zorder>labelValues</zorder>
  <zorder>verticalSlider</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>labelMassValue</zorder>
  <zorder>labelMass</zorder>
  <zorder>pushButtonMinAcoustic</zorder>
  <zorder>pushButtonMaxAcoustic</zorder>
  <zorder>pushButtonMaxOptical</zorder>
  <zorder>labelNameWindow</zorder>
  <zorder>labelW0</zorder>
  <zorder>labelChastota</zorder>
  <zorder>labelHuyeybl</zorder>
  <zorder>labelAcMaxFormula</zorder>
  <zorder>labelOptMinFormula</zorder>
  <zorder>labelOptMaxFormula</zorder>
  <zorder>labelHuyeybl_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>back</sender>
   <signal>clicked()</signal>
   <receiver>DemoWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
